src/components/organisms/StartProjectPane/StartProjectPage.tsx

```diff
@@ -15,6 +16,10 @@ const StartProjectPage = () => {
     dispatch(openFolderExplorer());
   };
 
+  const handleGitCloneRepo = () => {
+    // dispatch(openFolderExplorer());
```

Comment:
I think that we already have the functionality for cloning a git repo. We should put the `GitCloneModal` from `CreateProject.tsx` inside `App.tsx` and open/close it with the help of redux